lib/ansible/modules/network/nxos/nxos_logging.py

```diff
@@ -111,7 +147,26 @@
   nxos_logging:
     interface: mgmt0
     state: present
-
+- name: Purge configuration not what was just configured
```

Comment:
I found this description a bit confusing.  Perhaps the following wording instead?

`- name: Purge nxos_logging configuration not managed by this playbook`